javascript.js

```diff
@@ -23,3 +25,270 @@ document.querySelectorAll('.nav-item').forEach((n) => n.addEventListener('click'
+  document.querySelector('#Portfolio').removeAttribute('style', 'filter: blur(5px)');
+  document.querySelector('#About').removeAttribute('style', 'filter: blur(5px)');
+  document.querySelector('#Contact').removeAttribute('style', 'filter: blur(5px)');
+  document.querySelector('footer').removeAttribute('style', 'filter: blur(5px)');
+}
+
+// Dynamic content for main featured project card
+const main_project_container = document.querySelector('.featured_project');
+const main_card_content = document.querySelector('.main_card');
+
+const main_card_h3 = document.createElement('h3');
+main_card_h3.textContent = main_project.title;
+
+const main_card_p = document.createElement('p');
+main_card_p.textContent = main_project.description;
+
+const main_card_button = document.createElement('a');
+main_card_button.textContent = 'See Project';
+main_card_button.className = 'project-button';
+
+const main_card_tags = document.createElement('ul');
+main_card_tags.id = 'tags_featured';
+
+let main_card_tag_item = document.createElement('li');
+main_card_tag_item.textContent = main_project.features[0];
+main_card_tags.appendChild(main_card_tag_item);
+main_card_tag_item = document.createElement('li');
+main_card_tag_item.textContent = main_project.features[1];
+main_card_tags.appendChild(main_card_tag_item);
+main_card_tag_item = document.createElement('li');
+main_card_tag_item.textContent = main_project.features[2];
+main_card_tags.appendChild(main_card_tag_item);
+
+main_card_content.appendChild(main_card_h3);
+main_card_content.appendChild(main_card_p);
+main_card_content.appendChild(main_card_tags);
+main_card_content.appendChild(main_card_button);
+
+const main_project_img = document.createElement('img');
+main_project_img.id = 'featured_img';
+main_project_img.setAttribute('src', 'images/Img_Placeholder.png');
+
+main_project_container.appendChild(main_card_content);
+main_project_container.appendChild(main_project_img);
+
+// main card pop up action
+main_card_button.addEventListener('click', () => {
+  const popup = document.createElement('div');
+  popup.className = 'popup_container';
+  popup.classList.add('text_styles');
+
+  const popup_header = document.createElement('div');
+  popup_header.className = 'popup_header';
+  popup_header.appendChild(main_card_h3);
+  popup_header.appendChild(main_card_tags);
+
+  const popup_img = document.createElement('img');
+  popup_img.setAttribute('src', 'images/Img_Placeholder.png');
+
+  const live_button = document.createElement('a');
+  live_button.textContent = ' See Live';
+  live_button.className = 'project-button';
+
+  const source_button = document.createElement('a');
+  source_button.textContent = 'See Source';
+  source_button.className = 'project-button';
+
+  const see_live_icon = document.createElement('img');
+  see_live_icon.setAttribute('src', 'images/Icon-see live.svg'); see_live_icon.setAttribute('style', 'width: 16px; margin-left: 10px');
```

Comment:
- [ ] it would be better to make sure that the pop-up window is responsive on the mobiles screens as you can see in the picture below the buttons are overflowing please make sure to fix that 
<img width="355" alt="Screenshot 2023-03-09 at 12 16 31" src="https://user-images.githubusercontent.com/61790587/224009387-95ac0fd1-7c3a-42f2-9590-e158e7271374.png">